src/graphql/db_management.rs

```diff
@@ -1,13 +1,50 @@
 use std::sync::Arc;
 
-use async_graphql::{Context, Object, Result};
+use async_graphql::{Context, Object, Result, SimpleObject};
+use chrono::{DateTime, Utc};
 use review_database::{Store, backup};
 use tokio::sync::RwLock;
 use tracing::info;
 
 use super::{Role, RoleGuard};
 use crate::info_with_username;
 
+#[derive(SimpleObject)]
+pub struct BackupInfo {
+    pub id: u32,
+    pub timestamp: DateTime<Utc>,
+    pub size: u64,
+}
+
+#[derive(Default)]
+pub(super) struct DbManagementQuery;
+
+#[Object]
+impl DbManagementQuery {
+    #[graphql(guard = "RoleGuard::new(Role::SystemAdministrator)
+        .or(RoleGuard::new(Role::SecurityAdministrator))")]
+    async fn backups(&self, ctx: &Context<'_>) -> Result<Vec<BackupInfo>> {
+        let store = ctx.data::<Arc<RwLock<Store>>>()?;
+        info_with_username!(ctx, "Database backup list is being fetched");
+        let backup_infos = backup::list(store).await?;
+
+        // Convert from review_database::backup::BackupInfo to our GraphQL BackupInfo
+        // Sort by id in descending order (latest first)
+        let mut result: Vec<BackupInfo> = backup_infos
+            .into_iter()
+            .map(|info| BackupInfo {
+                id: info.id,
+                timestamp: info.timestamp,
+                size: info.size,
+            })
+            .collect();
+
+        result.sort_by(|a, b| b.id.cmp(&a.id));
```

Comment:
@henry0715-dev, could you explain why you chose to order by `id`? Ordering by `timestamp` seems more appropriate to me.